cdk/Makefile

```diff
@@ -19,3 +19,6 @@ create_grants:
 	
 revoke_grants:
 	npx ts-node bin/revoke_grants.ts
+
+clean:
```

Comment:
`make clean` is a common enough target that I think we could trigger muscle memory.

Let's call this something more explicit -- `empty_bucket` or `clear_contents` or something?